.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 npm-debug.log
-
+coverage
```

Comment:
Also, I thought we can reduce module size by only including required files and no tests. That's why I split gitignore and npmignore